kalite/distributed/static/css/distributed/bootstrap-overrides.css

```diff
@@ -436,7 +436,7 @@ footer {
 }
 
 @media (max-width: 454px) {
-    .form-control {
+    #search_box {
```

Comment:
I assume these changes have been made to only apply to the search box, rather than to all .form-control elements, is this because it was interfering with the styling of the coach report drop downs?